### ISSUE TITLE
Please fix

### ISSUE THREAD

**Issue by Rafapp** [Rha02/yousuf#22]
![image](https://github.com/Rha02/yousuf/assets/38381290/f66bf4e0-f1f7-4562-944c-febcfcb2c68b)

**Comment by Rha02** [comment]
I am on it

**Comment by Rha02** [comment]
@kevinboc 
I am assigning this issue to you. Take care of it

**Comment by kevinboc** [comment]
Rafa will be joining our senior project? I'll fix it right away

**Comment by Rafapp** [comment]
Yes, I'm the PM and CEO now in case you didn't get the memo. Please fix this stat. Thanks.
